### DISCUSSION TITLE
脚本网络代理

### DISCUSSION THREAD

**Discussion by hmgdd101** [the1812/Bilibili-Evolved#3464] [功能建议 / Ideas]
B站用翻墙的话会很卡，但是不翻墙脚本功能有问题，能不能给脚本加个使用代理的功能。
就比如我clash或v2ray在后台挂着不翻墙，脚本流量走clash和v2ray，这样就能连上jsdriver和gayhub了

**Comment by the1812** [comment]
不用翻 b 站啊，代理用 gfwlist 再额外加个 jsdelivr 就行了

**Reply by hmgdd101** [reply]
额，请问，怎么操作呢

**Reply by the1812** [reply]
https://github.com/FelisCatus/SwitchyOmega/wiki/GFWList

**Reply by hmgdd101** [reply]
今天出门差点没被热翻，微信请你吃个西瓜

**Comment by RingingVolcano** [comment]
代理软件设置好排除就行了
